src/Common/Select.php

```diff
@@ -612,12 +612,14 @@ public function groupBy(array $spec)
      *
      * @param string $cond The HAVING condition.
      *
+     * @param array ...$bind arguments to bind to placeholders
```

Comment:
another thought: the code for WhereInterface is duplicated across 3 classes. Would it be better to extract 'where()` and `orWhere()` into a WhereTrait to remove the duplication and simplify maintenance?